SharePointFramework/PortfolioWebParts/src/webparts/projectList/index.ts

```diff
@@ -6,16 +6,38 @@ import {
 import { IProjectListProps, ProjectList } from 'components/ProjectList'
 import * as strings from 'PortfolioWebPartsStrings'
 import { BasePortfolioWebPart } from 'webparts/@basePortfolioWebPart'
+import { Session, ITaxonomySession } from '@pnp/sp-taxonomy'
+import * as _ from 'lodash'
 
 export default class ProjectListWebPart extends BasePortfolioWebPart<IProjectListProps> {
   public render(): void {
+    console.log(this.properties);
     this.renderComponent<IProjectListProps>(ProjectList)
   }
 
+  private async _setupTaxonomy() {
+    const taxonomySession: ITaxonomySession = new Session()
+    const termSetIds = this.properties.termSetIds ? this.properties.termSetIds.split(',') : []
+    const termSetArray = []
+    const termStore = await taxonomySession.getDefaultSiteCollectionTermStore()
+    for (const termSetId of termSetIds) {
```

Comment:
How does this for loop work? `termSetIds` is just a **string**, not an array.

I see now that you're splitting it, but how does it work to have several term set IDs? What term set is this? Maybe use a more descriptive property name?

Reply:
Iirc:
If the portfolio has several term sets (Bygg, anlegg, program, etc.) in the same portfolio, we have to retrieve them all in order to give the cards the appropriate color (green/blue). 

It's just a string in the property pane, so we just have to instruct the admins to add Term sets as "ID1,ID2," which is why we pluralised the name, and are looping here.

Reply:
Gotcha!